yb-voyager/cmd/common.go

```diff
@@ -1049,6 +1049,13 @@ func (ar *AssessmentReport) GetClusterSizingRecommendation() string {
 
 // ==========================================================================
 
+func shouldSendCallhome() bool {
+	//here checking the startTime is initialised or not as can be seen in root.go we
+	//initialise startTime once we have logging setup after all initial checks
```

Comment:
nit: let's also add comments based on your findings for why uninitialized startTime in payload causes issues in callhome server?